GUI/src/javafxgui/JavaFXComponents/GUIView.java

```diff
@@ -20,14 +19,10 @@ public GUIView(Scene scene) {
         this.controller = null;
     }
 
-    public void setController(Controller controller) {
+    public void setController(GUIAppController controller) {
```

Comment:
No I can't as the GUIAppController uses DIP and is therefore constructed with the GUIView.  Because of this the GUIView can't be constructed with the Controller..
e.g. GUIAppController = new GUIAPPController( new GUIView(...), ...)